packages/amesos2/src/Amesos2_MUMPS_decl.hpp

```diff
@@ -205,6 +205,7 @@ class MUMPS : public SolverCore<Amesos2::MUMPS, Matrix, Vector>
 
   bool MUMPS_MATRIX_LOAD;
   bool MUMPS_STRUCT;
+  mutable bool MUMPS_MATRIX_LOAD_PREORDERING;
```

Comment:
"const is a lie"

Reply:
@mhoemmen Thanks for pointing this out. 
To 1: I just added another subtest wich `TEUCHOS_UNIT_TEST_TEMPLATE_3_INSTANT( MUMPS, SolveTwice, SCALAR, LO, GO )` and not a whole separated test. I assume that the whole test was protected before.

Reply:
@chochmuth You could try explicitly disabling Tpetra (`Trilinos_ENABLE_Tpetra:BOOL=OFF`) and enabling Epetra and Amesos2, just to see if it builds.  Amesos2 does claim to support this use case, so it's important to test.